source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/organizations/common/form.tsx

```diff
@@ -0,0 +1,279 @@
+              'admin.settings.organizations.add'
+            )
+          }
+        </h2>
+        <div className='flex w-60'>
+          <form data-test-form className='ui form flex-grow'>
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.name')}</label>
+              <input
+                data-test-org-name
+                type='text'
+                value={name || ''}
+                onChange={mut('name')} />
+            </div>
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.owner')}</label>
+              <div className='w-100 thin-bottom-border'>
+                <Dropdown
+                  className='w-100 no-borders'
+                  data-test-org-owner
+                  text={OwnerName}
+                >
+                  <Dropdown.Menu>
+                    {
+                      users.map((u, i) => {
+                        const { name: fullName } = attributesFor(u);
+                        return <Dropdown.Item key={i} text={fullName} onClick={this.ownerSelection(u)} />;
+                      })
+                    }
+                  </Dropdown.Menu>
+
+                </Dropdown>
+              </div>
+              <div className='error'>{ownerError}</div>
+            </div>
+
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.websiteURL')}</label>
+              <input
+                data-test-org-url
+                type='text'
+                value={websiteUrl || ''}
+                onChange={mut('websiteUrl')} />
+            </div>
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.buildEngineURL')}</label>
+              <input
+                data-test-build-engine-url
+                type='text'
+                value={buildEngineUrl || ''}
+                onChange={mut('buildEngineUrl')}
+              />
+            </div>
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.accessToken')}</label>
+              <input
+                data-test-build-engine-access-token
+                type='text'
+                value={buildEngineApiAccessToken || ''}
+                onChange={mut('buildEngineApiAccessToken')}
+              />
+            </div>
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.logoURL')}</label>
+              <input
+                data-test-logo-url
+                type='text'
+                value={logoUrl || ''}
+                onChange={mut('logoUrl')}
+              />
+            </div>
+            <div className='flex m-b-xl'>
+              <div className='flex-row align-items-center'>
+                <div className='p-r-lg'>
+                  <h3>
+                    {t('admin.settings.organizations.publicByDefault')}
```

Comment:
do these keys really need to be different from the user-facing translation strings?

Reply:
I dont understand which translation are you refering too?

Reply:
stuff that would be on the user-facing org settings pages